hack/lib/util.sh

```diff
@@ -14,6 +14,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+function kube::util::sourced_variable_ {
```

Comment:
Why the trailing dash?

Reply:
I was thinking that this function would be "mostly-internal" and not part of the API (even though we want to namespace it). I can remove the dash if you like.

Reply:
Removed the dash.